### ISSUE TITLE
Accessibility Issue: Separator not visible in Windows High Contrast modes

### ISSUE THREAD

**Issue by dipgupta-msft** [microsoft/fluentui#16193]
### Environment Information

- **Package version(s)**: ^7.150.0
- **Browser and OS versions**: Edge + Windows
### Describe the issue:

The actual separator is not visible in any of the windows high contrast modes.

### Please provide a reproduction of the issue in a codepen:
https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/fabric-website-resources/dist/index.html#/examples/separator

###### High Contrast White:
![image](https://user-images.githubusercontent.com/68835554/101674480-9250ef00-3a7e-11eb-8ae8-d85a2cbd3f5d.png)

###### High Contrast Black:
![image](https://user-images.githubusercontent.com/68835554/101674582-aeed2700-3a7e-11eb-81dc-6fb7f3f478f4.png)

#### Actual behavior:
Separator is not visible in any windows high contrast modes.

#### Expected behavior:

The separator should be visible in all windows high contrast modes.

**Comment by ecraig12345** [comment]
Thanks for reporting this. It looks like the issue is that we need to add high contrast overrides for Separator's `backgroundColor` in these two places. The override color should be `WindowText`.
https://github.com/microsoft/fluentui/blob/859fe008176c28a86b24d02e2d1bc1fe3d918752/packages/react-internal/src/components/Separator/Separator.styles.ts#L41
https://github.com/microsoft/fluentui/blob/859fe008176c28a86b24d02e2d1bc1fe3d918752/packages/react-internal/src/components/Separator/Separator.styles.ts#L57

@andrefcdias could you help with this one?
